debug_toolbar/panels/sql/panel.py

```diff
@@ -48,6 +47,33 @@ def get_transaction_status_display(vendor, level):
     return choices.get(level)
 
 
+def _similar_query_key(query):
+    return query["raw_sql"]
+
+
+def _duplicate_query_key(query):
+    raw_params = () if query["raw_params"] is None else tuple(query["raw_params"])
+    # repr() avoids problems because of unhashable types
+    # (e.g. lists) when used as dictionary keys.
+    # https://github.com/jazzband/django-debug-toolbar/issues/1091
+    return (query["raw_sql"], repr(raw_params))
+
+
+def _process_query_groups(query_groups, databases, colors, name):
+    counts = defaultdict(int)
+    for (alias, key), query_group in query_groups.items():
+        count = len(query_group)
+        # Queries are similar / duplicates only if there are at least 2 of them.
+        if count > 2:
```

Comment:
@living180 Should it be 
```python
if count > 1:
```
as per comment:
> if there are at least 2 of them.

Reply:
Good catch! My refactor accidentally changed a `>=` to a `>`.  Fixed, and added some tests which would catch this.